runtime/doc/userfunc.txt

```diff
@@ -0,0 +1,433 @@
+*userfunc.txt*	Nvim
+
+
+		  VIM REFERENCE MANUAL	  by Bram Moolenaar
+
+
+Defining and using functions.
+
+This is introduced in section |41.7| of the user manual.
+
+				      Type |gO| to see the table of contents.
+
+1. Defining a fuction			|define-function|
+2. Calling a fuction			|:call|
+3. Automatically loading functions	|autoload-functions|
```

Comment:
```suggestion
```

`gO` makes this redundant

Reply:
Looks like the same should be done for `builtin.txt` (and maybe some of the `pi_*` and `usr_*` stuff too, but I wouldn't bother with those here)